apps/ui/src/router.ts

```diff
@@ -43,7 +42,6 @@ const routes: any[] = [
       { path: '', name: 'space-overview', component: SpaceOverview },
       { path: 'proposals', name: 'space-proposals', component: SpaceProposals },
       { path: 'search', name: 'space-search', component: SpaceSearch },
-      { path: 'settings', name: 'space-settings', component: SpaceSettings },
```

Comment:
There won't be a way for others to view settings now.? it was easy to check/debug space settings

Reply:
Yeah, that is one concern I have as well, but this wasn’t publicly visible (but helpful for debugging). I wanted to make it visible for others, but we didn’t do it because it was supposed to change with new Edit Setting page, but looks like it’s still issue as now normal users can’t see it, and we won’t be able to use it for debugging as well (unless we pretend to be the controller).

@bonustrack

Reply:
This page should be visible for anyone (even unlogged) just not accessible from the sidebar if you are not logged with the controller address. I would also use "/settings" instead of "/edit-settings" route for that page.

Reply:
Right now this page just displays a message if you are not controller, what should we do if non-controller opens it?

Should we just block save actions? Or have special read-only mode (because there you can add/edit/remove strategies which would be accessible to non-controller), I guess if it's hidden by default just disabling save action might be enough.